City-of-Helsinki/drupal-helfi-etusivu
# UHF-7071 main news feed upgrades

# [UHF-7071](https://helsinkisolutionoffice.atlassian.net/browse/UHF-7071)


Migrations set all news items to be promoted. That's not what is wanted.

## What was done


Update hook which sets all news to not promoted.

## How to install
Use database which already has bunch of news items on multiple languages (marked as promoted on multiple languages, you can check `/{en or fi or sv}/admin/ordered-news` page)

* `git checkout dev`
* `make fresh`
* Check that you have the news items (`fi/admin/ordered-news`).
  * if not, promote some on multiple languages 
* `git pull origin UHF-7071_main_news_feed_upgrades`
* `make drush-updb` (if using prod db, this will take looooooong time)
* Run `make drush-cr`

## How to test

* after updb, go to `fi/admin/ordered-news`
  * You should not see any promoted news s
